File: navigation/safe-args-generator/src/tests/test-data/expected/nav_generator_test/java/nested_overridden_action/SettingsFragmentDirections.java
package foo.flavor;

import android.support.annotation.NonNull;
import foo.SettingsDirections;

public class SettingsFragmentDirections {
    private SettingsFragmentDirections() {
    }

    @NonNull
    public static SettingsDirections.Exit exit() {
        return SettingsDirections.exit();
    }
}

File: navigation/safe-args-generator/src/tests/test-data/expected/nav_generator_test/kotlin/nested_overridden_action/SettingsFragmentDirections.kt
package foo.flavor

import androidx.navigation.NavDirections
import foo.SettingsDirections
import kotlin.String

class SettingsFragmentDirections private constructor() {
    companion object {
        fun exit(exitReason: String = "DEFAULT"): NavDirections =
                SettingsDirections.exit(exitReason)
    }
}
